index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="  https://developers.google.com/fonts/docs/getting_started">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="logo">
+            <a href="#">Welcome</a>         
+        </nav>
+        <nav class="union-one">
+            <a>
+                <img src="./images/Icon - Menu.png" alt="">
+            </a>
+        </nav>
+    </header>
+    <section id="headline-section">
+        <div class="headline-container">
+            <h1>Hey there. I'm Mariane <br>
+                <span>I'm a software developer </span></h1>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hestiate to contact me.
+                </p>
+// just added the comments to save and modfy 
+                <ul>
```

Comment:
Please let's remove this text as it is not part of the Figma design and it is important that we stick to the design as much as possible

![image](https://user-images.githubusercontent.com/78251927/206823639-0e59f8a3-51cc-4d18-a2c0-9243acfa9661.png)